requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy
 torch
-normflows
+normflows @ git+https://github.com/VincentStimper/normalizing-flows.git
```

Comment:
I need the latest version of normflows here so that I can use the HMC gradient clipping